Sources/Sentry/SentryOptions.m

```diff
@@ -12,7 +12,7 @@ @implementation SentryOptions
 {
     return @[
         @"SentryCrashIntegration", @"SentryAutoBreadcrumbTrackingIntegration",
-        @"SentryAutoSessionTrackingIntegration"
+        @"SentryAutoSessionTrackingIntegration", @"SentryOutOfMemoryTrackingIntegration"
```

Comment:
created a Flutter task for that https://github.com/getsentry/sentry-dart/issues/379
@jennmueng you could create for RN/Cordova too, so we don't forget, thanks